src/addagent/tests/AddAgentWrapper.cpp

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
All these tests should be placed under the current tests schema located in `src/unit_tests`
following the guidelines provided in the documentation
making sure that these new tests are executed with the rest

Reply:
+1 to the location of the test. 

There's a folder `os_auth` that contains tests for the `validate.c` file in `addagents` folder. I don't know why that is the only case where the unit test folder is not the same as the one in src, so I would create a folder `src/unit_test/manage_keys` to add tests for `k_import` so it would be easiest if someone wants to add tests for others functions like `k_extract` and `k_bulkload`. 
To be discussed with the team